src/docs/feature-flags/flagpole.mdx

```diff
@@ -102,6 +102,50 @@ Each evaluation context is built up in stages via a [`ContextBuilder`](https://g
 
 Each context builder consists of a list of context transformers, which are responsible for creating individual slices of the larger evaluation context.
 
+Here are some common properties we surface via our Sentry and GetSentry context builders that you can use in your conditions:
+
+**Organization Context Properties**
+
+`organization_id` [int]
+
+: The organization's ID
+
+`organization_slug` [str]
+
+: The organization's slug
+
+`organization_name` [str]
+
+: The organization's name
+
+`organization_is-early-adopter` [bool]
+
+: Whether or not the organization has the `early_adopter` flag enabled
+
+**Project Context Properties (If checking a project feature)**
+
+`project_id` [int]
+
+: The project's ID
+
+`project_slug` [str]
+
+: The project's slug
+
+**User Context Properties (If checking a flag with an actor)**
+
+`user_id` [int]
+
+: The active user's ID
```

Comment:
Should we include `user_email` with the caveat of only using sentry.io email addresses?

Reply:
Good point, I'll add this to the docs.